File: programacao/c/respostas/operacoes/40.c
#include <stdio.h>

int main() {
    int n, i, inicio;

    printf("Digite um numero inteiro positivo n: ");
    scanf("%d", &n);

    inicio = n * n - (n - 1);

    printf("Os %d numeros impares consecutivos cuja soma e %d^3:\n", n, n);
    for (i = 0; i < n; i++) {
        printf("%d ", inicio + 2 * i);
    }
    printf("\n");

    return 0;
}

File: programacao/python/respostas/operacoes/40.py
n = int(input("Numero inteiro positivo n: "))
inicio = n * n - (n - 1)

print(f"Os {n} numeros impares consecutivos cuja soma e {n}^3:")
for i in range(n):
    print(inicio + 2 * i, end=" ")
print()
